sources/Tests/tests/ECS/ecs.cpp

```diff
@@ -228,3 +229,135 @@ TEST_CASE("game_events_handling", "[ecs]")
   gameWorld->emitEvent(TestEvent{20});
   REQUIRE(listener->getLastMessageCode() == 10);
 }
+
+TEST_CASE("game_objects_transform_components_management", "[ecs]")
```

Comment:
It is not a test for transform component. It is the test for iteration over game objects. So it will be better to set name like `game_objects_iteration`.